src/main/java/seedu/address/model/SalesBook.java

```diff
@@ -113,7 +114,10 @@ public void sortRecord() {
         record.sort();
     }
 
-    //// util methods
```

Comment:
Could you restore this line? to indicate the start of the util methods.

Reply:
Sure

Reply:
It should be restored below the setSalesRecordEntry method.